lib/utils.pm

```diff
@@ -1015,9 +1015,9 @@ sub validatelr {
     elsif (check_var('DISTRI', 'sle')) {
         # SLES12 does not have 'SLES12-Source-Pool' SCC channel
         unless (($version eq "12") and ($product_channel eq "Source-Pool")) {
+            my $distri = uc(get_var('DISTRI'));    # Capital letter, in this case SLE
             my $cmd
-              = "zypper lr --uri | awk -F \'|\' -v OFS=\' \' \'{ print \$2,\$3,\$4,\$NF }\' | tr -s \' \' | grep --color \"$product$version\[\[:alnum:\]\[:punct:\]\]*-*$product_channel $product$version\[\[:alnum:\]\[:punct:\]\[:space:\]\]*-*$product_channel $enabled_repo $uri\"";
-            run_scripted_command_slow($cmd, slow_type => 2);
```

Comment:
Why have we removed run_scripted_command_slow? with this code we do not use $cmd variable at all. Obviously, test run succeeds if we do not execute the command =).
I also believe that we can do parsing in perl instead of this monster.

Reply:
Thanks. That was a mistake. I'll rerun verifications.